index.bs

```diff
@@ -298,6 +298,20 @@ interface IntersectionObserver {
 		passed to the {{IntersectionObserver}} constructor.  If no
 		{{IntersectionObserverInit/scrollMargin}} was passed to the {{IntersectionObserver}}
 		constructor, the value of this attribute is "0px 0px 0px 0px".
+
```

Comment:
Sorry, my suggestion was not quite correct. The goal is to make sure that all the language describing how `scrollMargin` is applied is reachable from the algorithm description for "Compute the Intersection of a Target Element and the Root". That algorithm description currently refers to the `[[scrollMargin]]` slot, as distinct from the `scrollMargin` argument to the IntersectionObserver constructor.

I think instead it would make sense to put these two paragraphs back where they were, but add a line like 'To &lt;dfn&gt;apply scroll margin to a scrollport&lt;/dfn&gt;' above it. You should also move lines 291-292 ("&lt;b&gt;These offsets are only applied...") to the same place; and then refer to this new definition in the algorithm description for "Compute the Intersection of a Target Element and the Root", instead of referring directly to the `[[scrollMargin]]` slot.

Hopefully this makes sense...


So I think these two paragraphs should be moved to the place where the `[[scrollMargin]]` slot is defined; and the text on lines 291-292 ("These offsets are only applied when handling...") should also be moved there.

Reply:
Let me know if this looks better.